install.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+
+
+[ ! -d ~/.tldr ] && printf "tldr seems to have already been previously installed.\nDelete the \'$HOME/.tldr\' directory first before relaunching this script\n" && exit 1 || mkdir -p ~/.tldr/cache
+
+download() {
+  hash curl 2>/dev/null && curl -sf -o "$1" "$2" || { hash wget 2>/dev/null && wget -qO "$1" "$2"; } || echo "Fail to download $2" && exit 1
+}
+
+# Download tldr.sh and index.json
+download ~/.tldr/tldr.sh https://raw.githubusercontent.com/raylee/tldr/master/tldr.sh
+download ~/.tldr/cache/index.json http://tldr-pages.github.io/assets/index.json
+
+# Add alias to rc files
+add_alias() {
```

Comment:
I'm uncomfortable with this. Another goal for this script was such that it could be included in distros directly with no more dependencies than the core files. Part of that means being a good citizen with the standards they ask, which includes not automatically modifying the user's profile files. So I think this belongs either in the manual instructions or as an 'install' target on on the command line which does all the magic.

These aliases are unnecessary in the original instructions, as tldr is a command directly in their path:

`curl -o ~/bin/tldr https://raw.githubusercontent.com/raylee/tldr/master/tldr`